spec/mailers/previews/user_notifier_preview.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class UserNotifierPreview < ActionMailer::Preview
+  def send_followers_email
+    this_case = FactoryBot.create(:case, state_id: State.last.id + 1)
```

Comment:
Why do this here?  The database should be clean each time you run this.  Just create a state and assign that to be the state ID.

Reply:
I tried that too